WebDriverAgentLib/Routing/FBSession.m

```diff
@@ -179,7 +179,8 @@ - (XCUIApplication *)activeApplication
     XCUIApplicationState testedAppState = self.testedApplication.state;
     if (testedAppState >= XCUIApplicationStateRunningForeground) {
       if ([FBConfiguration shouldRespectSystemAlerts]
-          && [XCUIApplication.fb_systemApplication descendantsMatchingType:XCUIElementTypeAlert].count > 0) {
+          && ([XCUIApplication.fb_systemApplication descendantsMatchingType:XCUIElementTypeAlert].count > 0
+              || [XCUIApplication.fb_systemApplication fb_descendantsMatchingProperty:@"name" value:@"SBTransientOverlayWindow" partialSearch:false].count > 0)) {
```

Comment:
it makes sense to combine the condition in order to reduce the lookup time:

```
&& ([XCUIApplication.fb_systemApplication descendantsMatchingType:XCUIElementTypeAlert].count > 0
              || [XCUIApplication.fb_systemApplication fb_descendantsMatchingProperty:@"name" value:@"SBTransientOverlayWindow" partialSearch:false].count > 0))
``` 

change to 

```
  NSPredicate *searchPredicate = [NSPredicate predicateWithFormat:@"%K == %@ OR %K == %@", @"elementType", @(XCUIElementTypeAlert), @"identifier", @"SBTransientOverlayWindow"];
...
[[XCUIApplication.fb_systemApplication descendantsMatchingType:XCUIElementTypeAny] 
  matchingPredicate:searchPredicate].count > 0
```

Reply:
Looks good for the predicate.
I think leaving a comment like this PR's link will help future maintenance since the purchase's view is not easy to test out

Reply:
> it makes sense to combine the condition in order to reduce the lookup time:

Thank you for your suggestion. Applied as https://github.com/appium/WebDriverAgent/pull/946/commits/365280a45b1e50b2be0d20ac95635d6b6625bedf

> I think leaving a comment like this PR's link will help future maintenance since the purchase's view is not easy to test out

Thank you for your comment. Added a comment accordingly: https://github.com/appium/WebDriverAgent/pull/946/commits/024ea3729dd61a4304ada3f01d5d05401adcda73